ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_statistical.py

```diff
@@ -317,3 +337,36 @@ def test_bincount(
         weights=x[1],
         minlength=min_length,
     )
+
+
+# percentile
+@handle_test(
+    fn_tree="functional.ivy.experimental.percentile",
+    dtype_and_x=statistical_dtype_values(function="percentile"),
+    keep_dims=st.booleans(),
+    test_gradients=st.just(False),
+)
+def test_percentile(
+    *,
+    dtype_and_x,
+    keep_dims,
+    test_flags,
+    backend_fw,
+    fn_name,
+    on_device,
+    ground_truth_backend,
+):
+    input_dtype, x, axis, interpolation, q = dtype_and_x
+    helpers.test_function(
+        input_dtypes=input_dtype,
+        test_flags=test_flags,
+        ground_truth_backend=ground_truth_backend,
+        fw=backend_fw,
+        fn_name=fn_name,
+        on_device=on_device,
+        a=x[0],
+        q=q,
+        axis=axis,
+        interpolation=interpolation[0],
+        keepdims=keep_dims,
```

Comment:
The difference between the results seems to be very small. You could try setting the `rtol_` and/or `atol_` arguments of `test_function` to avoid getting the assertion error. My guess is that `atol=1e-04` will suffice.